Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,16 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  arr.sort(function(letter1, letter2) {
+    return letter1 > letter2;
+  });
+  const unique = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] !== arr[i + 1]) {
+      unique.push(arr[i]);
+    }
+  }
+  return unique;
```

Comment:
Not quite, if I run this code `uniqueValues` has the same exact letters and order as the original `values`. Perhaps re-think this. `filter` seems like it might be a suitable method for the job? Or perhaps `Set`?